# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/iproute2/hotfix-limits.patch.musl
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- iproute2-6.18.0/lib/utils_math.c.vanilla	2026-01-03 12:48:45.603955229 +0100
+++ iproute2-6.18.0/lib/utils_math.c	2026-01-03 12:48:52.081275261 +0100
@@ -5,6 +5,7 @@
 #include <string.h>
 #include <math.h>
 #include <asm/types.h>
+#include <limits.h>
 
 #include "utils.h"
 
--- iproute2-6.18.0/tc/tc_core.c.vanilla	2026-01-03 12:49:21.578436511 +0100
+++ iproute2-6.18.0/tc/tc_core.c	2026-01-03 12:49:29.598237715 +0100
@@ -15,6 +15,7 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <string.h>
+#include <limits.h>
 
 #include "utils.h"
 #include "tc_core.h"
